pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.ruff]
 
-select = ["D", "E", "F", "W", "C", "S", "I", "TCH"]
+select = ["D", "E", "F", "W", "C", "S", "I", "TCH", "SLOT", "RUF", "C90", "N"]
```

Comment:
@tisnik can we get a comment in here that indicates what each of these are?  or a link to the docs that describe them?  (both the existing ones and the ones you just added)

Reply:
perhaps you can do it as part of https://github.com/openshift/lightspeed-service/pull/216